glib/src/object.rs

```diff
@@ -3560,6 +3560,25 @@ impl<'a> BindingBuilder<'a> {
         }
     }
 
+    // rustdoc-stripper-ignore-next
+    /// Transform changed property values from the target object to the source object with the given closure.
+    pub fn transform_from_concrete<
```

Comment:
I would call this one `transform_from()` and the other one `transform_from_with_values()` or so

Reply:
All right!